src/index.tsx

```diff
@@ -1,6 +1,10 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
+import './styles/reset.css';
+import './styles/base.css';
+import './styles/form.css';
+import './styles/variables.css';
```

Comment:
```suggestion
import 'styles/reset.css';
import 'styles/base.css';
import 'styles/form.css';
import 'styles/variables.css';
```
Doesn't really matter here as we're in the root, but just as a rule of thumb try to favor absolute paths as it removes unecessary noise.

Reply:
Can do 👍

Reply:
FYI, github has a "commit suggestion" button.

Reply:
Yup, that's what I used 👍